Northstar.Client/mod/scripts/vscripts/ui/panel_mainmenu.nut

```diff
@@ -534,7 +520,7 @@ void function OnPlayFDButton_Activate( var button ) // repurposed for launching
 {
 	if ( !Hud_IsLocked( button ) )
 	{
-		SetConVarString( "communities_hostname", "" ) // disable communities due to crash exploits that are still possible through it
+		// SetConVarString( "communities_hostname", "" ) // disable communities due to crash exploits that are still possible through it
```

Comment:
... why comment this?

Reply:
I was going to move this around and forgot, should be done now